waterlog/src/Components/AlertBox.js

```diff
@@ -2,26 +2,26 @@ import React from 'react';
 
 function highStatus() {
 	return (
-		<svg width="15" height="15">
-			<rect x="0" y="0" width="15" height="15" style={{ fill: '#ff1744' }} />
+		<svg width="12" height="12">
+			<rect x="0" y="0" width="12" height="12" style={{ fill: '#ff1744' }} />
```

Comment:
You are repeating these styles. 
Please make use of css classes instead

Reply:
Working on it

Reply:
Please review i am done some changes